.env.development

```diff
@@ -1,5 +1,7 @@
 EXAMPLE_VAR=Example Value
 ACCESS_TOKEN_COOKIE_NAME=edx-jwt-cookie-header-payload
+ACCOUNT_PROFILE_URL=http://localhost:1995/u
```

Comment:
Please clarify why do our URL looks a little bit weird with the suffix `/u` ?

Reply:
I just brought the URL to one view, as in the platform: https://github.com/raccoongang/edx-platform/pull/2419/files#diff-aa3422bb9dd162d0f77ccc8e4f2bd3bfe204f1f3eeb6ecbbf99e7462cf16edb9R338. Should I change it without the suffix? frontend-app-profile has only this valid url: `/u/:username` and error url `/notfound`.

Reply:
I would prefer we have all URLs in a consistence way so our settings should be mapped to certain services. An appropriate URL should be built on the base_service_url + API URL as we usually do.

Reply:
Updated